src/screens/AnswerScreen/AnswerScreen.js

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable react-native/no-inline-styles */
+/* eslint-disable no-alert */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
I think we can remove this from here. Is it?

Reply:
It was added by the linter itself. These lines are disabling some rules of linter that are causing a change in code!